src/app/user/list/user-list.component.html

```diff
@@ -78,9 +78,9 @@ <h4 class="content-title">{{paginationConfig.totalItems | i18nPlural: {'=0': 'No
                         </div>
                         <div class="row">
                           <div class="card-metainfo col m6">
-                            <p>{{user.title}}</p>
+                            <p class="user-title">{{user.title}}</p>
                           </div>
-                          <div class="card-metainfo col m6">
+                          <div class="card-metainfo col m6 user-country">
```

Comment:
Is there a reason you're adding a `user-country` class to this element? Seems irrelevant to your changeset?